content/roadmap.md

```diff
@@ -2,21 +2,19 @@
 title: Roadmap
 ---
 
-The following is only a selection of some of the major features we plan to implement in the near future (6-12 months). To get a more complete overview of planned features and current work, see the issue trackers for the various repositories, for example, the [Jaeger backend](https://github.com/jaegertracing/jaeger/issues/).
+The following is only a selection of some of the major features we plan to implement, some of which are near term and some are longer term. We have tried to put these in rough priority as well as having a wishlist at the end. To get a more complete overview of planned features and current work, see the issue trackers for the various repositories, for example, the [Jaeger backend](https://github.com/jaegertracing/jaeger/issues/).
 
-## Integration with OpenTelemetry collector
+## Aggregated Trace Metrics (ATM)
 
-[OpenTelemetry collector](https://opentelemetry.io/docs/collector/getting-started/) is a vendor-agnostic service for receiving, processing and exporting telemetry data. We have decided to rebuild the Jaeger backed components (agent, collector, ingester, all-in-one) on top of OpenTelemetry collector which has several benefits:
+Aggregated trace metrics can be exported by the OpenTelemetry Collector [/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor) this allows metrics to be calculated off trace data and metrics to be exported from the Collector. We are adapting Jaeger to be able to read metrics from a Prometheus compatible backend, but additional backends may be supported. For additional information see [jaeger/pull/2946](https://github.com/jaegertracing/jaeger/pull/2946). 
```

Comment:
```suggestion
Aggregated trace metrics can be exported from the OpenTelemetry Collector by [spanmetricsprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor) that allows metrics to be calculated from trace data. We are adapting Jaeger to be able to read metrics from a Prometheus compatible backend, but additional backends may be supported. For additional information see [jaeger/pull/2946](https://github.com/jaegertracing/jaeger/pull/2946). 
```

Reply:
It's better to link to an issue rather than pull request: https://github.com/jaegertracing/jaeger-ui/issues/736

Reply:
That issue is for the UI, not the Otel processor. I linked to https://github.com/jaegertracing/jaeger/issues/2954 which is the master issue for the UI change.